src/citrine/resources/data_concepts.py

```diff
@@ -140,8 +140,6 @@ def _get_field(data, field: str):
             return data.get(field)
         elif isinstance(data, object):
             return getattr(data, field, None)
-        else:
-            TypeError("Expected data to be a dictionary or object, instead got {}".format(data))
```

Comment:
logically unreachable...the line before tests for object....everything is an object